### PR TITLE
[Common] (snippets) Map snippet to Office.AddinCommands.Source

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[ded0429](https://github.com/OfficeDev/office-js-docs-reference/pull/323/commits/ded04299bd33e9e96b0d5aed28431bdcd521322e)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 1 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[ded0429](https://github.com/OfficeDev/office-js-docs-reference/pull/323/commits/ded04299bd33e9e96b0d5aed28431bdcd521322e)_: 

:clock10: **Incremental building**: average incremental building time is _4 min(s) 56 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[ded0429](https://github.com/OfficeDev/office-js-docs-reference/pull/323/commits/ded04299bd33e9e96b0d5aed28431bdcd521322e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/code-snippets/office-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-addincommands-snippets/docs/code-snippets/office-snippets.yaml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.addincommands.event.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-addincommands-snippets/docs/docs-ref-autogen/office/office.addincommands.event.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.addincommands.source.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-addincommands-snippets/docs/docs-ref-autogen/office/office.addincommands.source.yml) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-custom-functions-runtime/custom-functions-runtime.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-addincommands-snippets/generate-docs/api-extractor-inputs-custom-functions-runtime/custom-functions-runtime.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-office-runtime/office-runtime.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-addincommands-snippets/generate-docs/api-extractor-inputs-office-runtime/office-runtime.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/json/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-addincommands-snippets/generate-docs/json/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/custom-functions-runtime.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-addincommands-snippets/generate-docs/script-inputs/custom-functions-runtime.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/local-repo-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-addincommands-snippets/generate-docs/script-inputs/local-repo-snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/office-runtime.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-addincommands-snippets/generate-docs/script-inputs/office-runtime.d.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C14%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201902141908088426-323%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=8WD%2Bicmf4KjQqJBHOmZy1af1474TfyLfIgpYA3VYqdo%3D&st=2019-02-14T19%3A07%3A21Z&se=2019-03-17T19%3A12%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: generate-docs/api-extractor-inputs-custom-functions-runtime/custom-functions-runtime.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for Custom Functions 1.4
+// Type definitions for non-npm package Custom Functions 1.4
```
Comment (Rick-Kirkham) [inline generate-docs/api-extractor-inputs-custom-functions-runtime/custom-functions-runtime.d.ts]: Something similar to "Type definitions for non-npm package Custom Functions 1.4" appears several places in this PR. Can this be clarified? Does it mean definitions for CF that are not in any package at all? Or does it mean they are in a package, but not an npm package? Or does it mean some third thing?

Comment (ElizabethSamuel-MSFT) [reply generate-docs/api-extractor-inputs-custom-functions-runtime/custom-functions-runtime.d.ts]: This change was made by DefinitelyTyped folks.

**Pull request merged**